conf/etc/common/README.md

```diff
@@ -0,0 +1 @@
+## README for common modules
```

Comment:
Can add more meaningful text -
Placeholder for common configuration files in v1 and v2.

Reply:
Done